src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,6 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_MATRIC = new Prefix("m/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
+    public static final Prefix PREFIX_MODULE = new Prefix("m/");
```

Comment:
Just leaving a note here:

Check if this has been defined the same way in #108.

Reply:
They've called it PREFIX_MOD_CODE and I called it PREFIX_MODULE. Can we keep both or must change? If so, I can change but more convenient to keep :P

Reply:
Keep this for now! We'll refactor this out later!